nautobot/core/settings.py

```diff
@@ -907,3 +907,6 @@ def UI_RACK_VIEW_TRUNCATE_FUNCTION(device_display_name):
     "TimeZoneSerializerField": {"type": "string"},
     "UUIDField": {"type": "string", "format": "uuid"},
 }
+
+# Turn on or off new ui feature
+ENABLE_NEW_UI_FEATURE = is_truthy(os.getenv("ENABLE_NEW_UI", "False"))
```

Comment:
Also we need to decide how/whether to document this setting and the associated change in behavior. I'm not sure a `.changed` entry is sufficient, it might be good to call this out in the release-note Release Overview section as well?